client/login/wp-login/index.jsx

```diff
@@ -138,6 +140,9 @@ class Login extends React.Component {
 		return (
 			<Main className="wp-login">
 				<PageViewTracker path="/login" title="Login" />
+
+				<GlobalNotices id="notices" notices={ notices.list } />
```

Comment:
This view can be rendered logged in or logged out. When logged in, `GlobalNotices` is rendered twice while it's intended to be rendered once.

This means any global notices and they're respective html are duplicated. They're stacked so it's mostly irrelevant, but will notice a darker shadow where they overlap.

I'm trying to find out why `GlobalNotices` is omitted from the logged-out layout. Including it there would mean we could remove this and save the duplication simply.